lib/erb_lint/cli.rb

```diff
@@ -88,44 +95,33 @@ def autocorrect?
 
     def run_with_corrections(runner, filename)
       file_content = File.read(filename)
-      processed_source = ERBLint::ProcessedSource.new(filename, file_content)
-      offenses = runner.run(processed_source)
+      offenses = []
 
-      if offenses.any? && autocorrect?
-        corrector = correct(processed_source, offenses)
-        processed_source = ERBLint::ProcessedSource.new(filename, corrector.corrected_content)
+      7.times do
```

Comment:
Why this iteration? why 7 times? I am lost with this.

Thanks!

Reply:
See #103.